apps/sms/style/sms.css

```diff
@@ -902,6 +902,18 @@ section[role="region"].new .article-list[data-type="list"]  li {
   pointer-events: none;
 }
 
+/* margin bottom adjustment for inline-block bdi element */
+.suggestion > .name {
+  margin-bottom: -0.5rem;
+}
+
+.suggestion > .name > bdi {
```

Comment:
Ah sorry for not being clear :) I mean  property reordering inside single CSS rule as we usually do, like https://github.com/necolas/idiomatic-css#format.

Regarding to ```separate css class for bdi that supports ellipsis``` I mean something like ```.can-have-ellipsis``` (bad name though) so that we can add to required ```<bdi>``` elements, but as I said I don't have strong opinion on that - so up to you :)